projects/control-service/projects/job-base-image-secure/README.md

```diff
@@ -0,0 +1,17 @@
+# Job base image
+
+Job base image is the container "base" image used when building per data job custom image during deployment.
+
+This directory provides the source of some base images for standard python versions.
+It's used by default installation of VDK and should also serve as an example on how operators of VDK can build their own base image.
+
+The current base image installs supporting libraries for Oracle database
```

Comment:
The README looks wrong. Do we plan to support Oracle libraries in view that now there's pure python library that can be used without native client installations required (https://python-oracledb.readthedocs.io) ?

Reply:
Correct, I will update it.